modules/nix/linux-builder.nix

```diff
@@ -20,6 +20,7 @@ let
     rm -rf $TMPDIR
     mkdir -p $TMPDIR
     trap "rm -rf $TMPDIR" EXIT
+    ${lib.optionalString cfg.ephemeral "rm -f ${cfg.workingDirectory}/nixos.qcow2"}
```

Comment:
```suggestion
    ${lib.optionalString cfg.ephemeral "rm ${cfg.workingDirectory}/{cfg.config.networking.hostname}.qcow2"}
```

Reply:
had to slightly change this but it's done.